### ISSUE TITLE
Groq API in PrivateGPT

### ISSUE THREAD

**Issue by psmukhopadhyay** [zylon-ai/private-gpt#1945]
Hello

I am a beginner-level user of PrivateGPT and set it for 'local' with LLM as mistral-7b-instruct-v0.2.Q4_K_M.gguf.

Please advise me how to add Groq (OpenAI compatible LLM service - https://console.groq.com/docs/quickstart) in PrivateGPT. I already have API key and other ID for Groq. Should I follow the included azopenai as model?

Regards

**Comment by psmukhopadhyay** [comment]
Thanks Ken :)

**Comment by nizar0x1f** [comment]
any updates guys?

**Comment by somashekhar161** [comment]
yes i was able to integrate it with fireworks hoping it'll be similar to groq

**Comment by somashekhar161** [comment]
> any updates guys?

https://github.com/zylon-ai/private-gpt/pull/2089
check this out added fireworks integration
